### ISSUE TITLE
Unable to install graphkernals in R

### ISSUE THREAD

**Issue by yasodajayaweera** [BorgwardtLab/graph-kernels#2]
I'm trying to install graphkernals package in R and I'm getting the following error. I'm running R 3.1.3 on a Windows 7 (x64) machine.

**Steps followed:**
1. Installed the packages Rcpp, RcppEigen and igraph
2. Downloaded _graphkernels_1.4.tar.gz_ file from https://cran.r-project.org/web/packages/graphkernels/index.html 
3. Ran the command _install.packages("D:/graph/graphkernels_1.4.tar.gz", repos = NULL, type = "source")_

**Error message:**
* installing *source* package 'graphkernels' ...
** package 'graphkernels' successfully unpacked and MD5 sums checked
** libs

*** arch - i386
Warning: running command 'make -f "Makevars.win" -f "C:/PROGRA~1/R/R-31~1.3/etc/i386/Makeconf" -f "C:/PROGRA~1/R/R-31~1.3/share/make/winshlib.mk" CXX='$(CXX1X) $(CXX1XSTD)' CXXFLAGS='$(CXX1XFLAGS)' CXXPICFLAGS='$(CXX1XPICFLAGS)' SHLIB_LDFLAGS='$(SHLIB_CXX1XLDFLAGS)' SHLIB_LD='$(SHLIB_CXX1XLD)' SHLIB="graphkernels.dll" OBJECTS="RcppExports.o graphKernels.o graphkernels_init.o"' had status 127
ERROR: compilation failed for package 'graphkernels'
* removing 'C:/Program Files/R/R-3.1.3/library/graphkernels'
Warning in install.packages :
  running command '"C:/PROGRA~1/R/R-31~1.3/bin/x64/R" CMD INSTALL -l "C:\Program Files\R\R-3.1.3\library" "D:/graph/graphkernels_1.4.tar.gz"' had status 1
Warning in install.packages :
  installation of package ‘D:/graph/graphkernels_1.4.tar.gz’ had non-zero exit status

I've tried installing graphkernals the same way in a Ubuntu machine as well. However I could not install it as the igraph package couldn't be installed in Ubuntu for some reason.

Can you please help me sort this. Thank you.

**Comment by mahito-sugiyama** [comment]
Thank you for your comment. I will check the issue.
For now, since the newest version is on CRAN, could you try to install the package directly from the CRAN by the following?

In R console:
`> install.packages("graphkernels", dependencies = TRUE)`

**Comment by yasodajayaweera** [comment]
@mahito-sugiyama Thank you for the response. 

I executed `install.packages()` as you suggested and got the following error. 
package ‘graphkernels’ is available as a source package but not as a binary

> Warning in install.packages :
>   package ‘graphkernels’ is not available (as a binary package for R version 3.1.3)
> 

I tried updating my R version but I couldn't update it due to [another error](https://github.com/talgalili/installr/issues/94). So I uninstalled R and installed the latest R version 3.4.2. Then I tried installing graphkernels package and it worked. 

Thank you.
